### ISSUE TITLE
The system cannot find the path specified.

### ISSUE THREAD

**Issue by horta** [joerick/cibuildwheel#141]
Repository: https://github.com/limix/pandas-plink/tree/develop

```
# appveyor.yml
environment:
  PKG_NAME: pandas_plink
  CIBW_TEST_COMMAND: "python -c \"import %PKG_NAME%; import sys; sys.exit(%PKG_NAME%.test())\""
  CIBW_SKIP: cp27-* cp33-* cp34-* cp35-*

install:
  - set PATH=C:\Python37;C:\Python37\Scripts;%PATH%
  - cmd: python -m pip install cibuildwheel --upgrade

build_script:
  - cd %APPVEYOR_BUILD_FOLDER%
  - powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.githubusercontent.com/horta/ci/master/appveyor.bat', 'appveyor.bat')" && appveyor.bat
  - cd %APPVEYOR_BUILD_FOLDER%
  - cmd: git clean -xdf
  - cmd: python -m cibuildwheel --output-dir wheelhouse
  - ps: >-
      if ($env:APPVEYOR_REPO_TAG -eq "true") {
        Invoke-Expression "python -m pip install twine"
        Invoke-Expression "python -m twine upload --skip-existing wheelhouse/*.whl"
      }

artifacts:
  - path: "wheelhouse\\*.whl"
    name: Wheels
```

**Error**

```
python -m cibuildwheel --output-dir wheelhouse
     _ _       _ _   _       _           _
 ___|_| |_ _ _|_| |_| |_ _ _| |_ ___ ___| |
|  _| | . | | | | | . | | | |   | -_| -_| |
|___|_|___|___|_|_|___|_____|_|_|___|___|_|
cibuildwheel version 0.11.0
Build options:
  platform: 'windows'
  before_build: None
  build_selector: BuildSelector('*' - 'cp27-* cp33-* cp34-* cp35-*')
  build_verbosity: 0
  environment: ParsedEnvironment([])
  output_dir: 'wheelhouse'
  project_dir: '.'
  test_command: 'python -c "import pandas_plink; import sys; sys.exit(pandas_plink.test())"'
  test_requires: []
Here we go!
+ python --version
The system cannot find the path specified.
Traceback (most recent call last):
  File "C:\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python37\lib\site-packages\cibuildwheel\__main__.py", line 222, in <module>
    main()
  File "C:\Python37\lib\site-packages\cibuildwheel\__main__.py", line 161, in main
    cibuildwheel.windows.build(**build_options)
  File "C:\Python37\lib\site-packages\cibuildwheel\windows.py", line 106, in build
    shell(['python', '--version'], env=env)
  File "C:\Python37\lib\site-packages\cibuildwheel\windows.py", line 74, in shell
    return subprocess.check_call(' '.join(args), env=env, cwd=cwd)
  File "C:\Python37\lib\subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cmd /E:ON /V:ON /C C:\Python37\lib\site-packages\cibuildwheel\resources\appveyor_run_with_env.cmd python --version' returned non-zero exit status 1.
```

**Comment by joerick** [comment]
ah, dang, that's a packaging problem with the latest release. I'll look into it...

**Comment by joerick** [comment]
Should be fixed in 0.11.1. Let me know if not!
